kin-sdk/src/androidTest/kotlin/kin/sdk/KinAccountIntegrationTest.kt

```diff
@@ -18,21 +17,25 @@ import java.math.BigDecimal
 import java.util.*
 import java.util.concurrent.CountDownLatch
 import java.util.concurrent.TimeUnit
+import kotlin.test.assertTrue
 import kotlin.test.fail
 
 @Suppress("FunctionName")
 class KinAccountIntegrationTest {
 
     private val appId = "1a2c"
+    private val fee: Int = 100
     private val appIdVersionPrefix = "1"
+    private val timeoutDurationSeconds : Long = 10
+    private val timeoutDurationSecondsLong : Long = 5
```

Comment:
should be the opposite I guess (long =10 sec)

Reply:
good catch, i meant 15 but ending with only 5...